src/project-css/database/migrations/2020_03_18_162037_add_iscms_to_collections.php

```diff
@@ -4,7 +4,7 @@
 use Illuminate\Database\Schema\Blueprint;
 use Illuminate\Database\Migrations\Migration;
 
-class CreateStopWordsTable extends Migration
+class AddIscmsToCollections extends Migration
 {
     /**
      * Run the migrations.
```

Comment:
Needs Proper Documentation: 
Use full PHP Documentation Standards.
Be clear on what the parameters of the function are as well as what the return values are supposed to be.